gremlinapi/__init__.py

```diff
@@ -150,17 +150,17 @@ def filter(self, record):
 _https_proxy = os.getenv("GREMLIN_HTTPS_PROXY", os.getenv("HTTPS_PROXY", None))
 
 
-GremlinAPIConfig.user = _api_user
-GremlinAPIConfig.password = _api_password
-GremlinAPIConfig.user_mfa_token_value = _api_user_mfa_token
-GremlinAPIConfig.base_uri = f"{_api_host}/{_api_version}"
-GremlinAPIConfig.company_name = _api_company
-GremlinAPIConfig.api_key = _api_key
-GremlinAPIConfig.bearer_token = _api_bearer_token
-GremlinAPIConfig.bearer_timestamp = _bearer_token_timestamp
-GremlinAPIConfig.max_bearer_interval = _max_bearer_interval
-GremlinAPIConfig.http_proxy = _http_proxy
-GremlinAPIConfig.https_proxy = _https_proxy
+GremlinAPIConfig.user = _api_user  # type: ignore
```

Comment:
I think all of these pre-configuration variables should be strings or ints.

Reply:
Will remediate, in scope for this PR